src/oas.md

```diff
@@ -2545,6 +2638,7 @@ ETag:
       schema:
         type: string
         pattern: ^"
+  example: xyzzx
```

Comment:
I don’t understand why the example does not match the pattern required by the schema.

Reply:
@ralfhandl I updated it with that approach and added a comment about the unusual quote use.... I am open to dropping this as too confusing but it's actually a semantically significant documentation use case.

Reply:
(I have no idea if anyone would _use_ this in an actual OAD, but it does tell you something useful).